src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.4.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.4.0
+ */
+class WP_Font_Library {
+
+	/**
+	 * Provide the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
```

Comment:
```suggestion
	 * Provides the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
```
- Method summaries should use a third-person singular verb per [the Inline Documentation Standards](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#language).

Reply:
In addition, this method is declared before the `$collections` property, and I believe this should instead be placed below property declarations.

Reply:
yes, will push a change to put the property first.

Reply:
Addressed in these two commits:

https://github.com/WordPress/wordpress-develop/pull/5285/commits/17c9fd335574f3e85df86e607d1d878531bb1c33
https://github.com/WordPress/wordpress-develop/pull/5285/commits/3021090f0c11781598d780e6997cb9bfb5ab12fb